code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -390,10 +390,19 @@
 									span_notice("You cut \the [src] apart with \the [W]."))
 				deconstruct(TRUE)
 				return
+		if(W.tool_behaviour == TOOL_WRENCH && anchorable) // The locker is open and we're using a wrench
+			if(isinspace() && !anchored)
+				return
+			set_anchored(!anchored)
+			W.play_tool_sound(src, 75)
+			user.balloon_alert_to_viewers("[anchored ? "anchored" : "unanchored"].")
```

Comment:
```suggestion
			user.balloon_alert_to_viewers("[anchored ? "anchored" : "unanchored"]")
```

Punctuation is only for `...` (to indicate actions being taken) and `!` (to indicate failure)

Reply:
This was automatically resolved when I committed your punctuation suggestion. I will work on moving wrench code to its own proc tomorrow.